### ISSUE TITLE
img alt text examples need more context

### ISSUE THREAD

**Issue by JohannesBeranek** [mdn/content#36050]
### MDN URL

https://developer.mozilla.org/en-US/docs/Web/HTML/Element/img

### What specific section or headline is this issue about?

alt attribute example

### What information was incorrect, unhelpful, or incomplete?

```html
<img alt="A Rockhopper Penguin is standing on a beach." src="penguin.jpg" />
```

### What did you expect to see?

A better example for what should be in an alt tag. I can't imagine a single situation where the provided alt text example would actually make sense.

You could for example do
```html
<img alt="down" src="down-arrow.jpg" />
```
(I deliberately didn't use "next" or something like that, because it could be dependent on the writing direction)

### MDN metadata

<details>
<summary>Page report details</summary>

* Folder: `en-us/web/html/element/img`
* MDN URL: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/img
* GitHub URL: https://github.com/mdn/content/blob/main/files/en-us/web/html/element/img/index.md
* Last commit: https://github.com/mdn/content/commit/75326725db2daa924618e58ae31a43345c7a16dc
* Document last modified: 2024-08-12T06:55:47.000Z

</details>

**Comment by Josh-Cena** [comment]
The alt text example accurately describes what you should write if you have an image of "A Rockhopper Penguin is standing on a beach". The alt text should be a 1-to-1 reflection of the image content, as if your browser can only display text.

**Comment by JohannesBeranek** [comment]
@Josh-Cena AFAIK the alt text should be what screen readers should read instead of the image. The text "A Rockhopper Penguin is standing on a beach." is not useful information for the visitor in any context, and giving an example like that leads to just people putting image descriptions into alt texts for purely decorative images, which is wrong.

**Comment by JohannesBeranek** [comment]
"... what the image is trying to communicate." is not the same as "... what you can see on the image." or "... what is depicted by the image."

If you used a right arrow image for a "next page" action, an alt text should NOT be `alt="right arrow"`, but `alt="next page"`.

**Comment by Josh-Cena** [comment]
Right, it should describe whatever you want the screen reader to announce. Imagine it in this context:

> Last week, I went on a trip to South Africa. I saw a lot of fascinating things, such as these:
>
> [Image: A Rockhopper penguin is standing on a rock.]

I don't see how your suggestion is incompatible with the current example: the alt text for a photo describes the information of what's in that photo, and the alt text for an icon describes the information of that icon.

**Comment by JohannesBeranek** [comment]
@Josh-Cena I get what you mean and understand it from the context you provided, but that's exactly my issue - that context is missing there. And while I don't have the visitor numbers of MDN, I would - from my 30 years of dev experience - assume that most people won't follow those links, but just take the examples as they are, without any further context provided by the links.

I'd be very happy to just add that little bit of context somehow, i.e. instead of having only the `<img ...>` tag in the DO and DON'T boxes, also add that little bit of text before it, and/or maybe a note like `A screen reader could read this as 'Last week, I went on a trip to South Africa. I saw a lot of fascinating things, such as these: image'`. People who've never done anything accessibility related wouldn't event think about screen readers usually, and without that context, I think that the examples, as they are, make for a bad introduction to alt texts.

**Comment by Josh-Cena** [comment]
Alright! More context to demonstrate user experience is welcome. If you want to add a paragraph about alt text for icons I'm open to it too.

cc @estelle

**Comment by Josh-Cena** [comment]
Also apparently rockhopper penguins do not live in South Africa as a natural habitat—they are from the southern islands of Atlantic and Indian oceans.

**Comment by JohannesBeranek** [comment]
I can provide "raw" texts, but I'm not natively English speaking unfortunately.

Should I create a PR to continue a discussion there, or just add something here?

**Comment by Josh-Cena** [comment]
I would prefer a PR, but whatever you feel comfortable with x
